.github/workflows/linters.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: CI
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    branches:
+      - main
+      - stable-*
+
+jobs:
+  linters:
+    uses: abikouo/github_actions/.github/workflows/tox-linters.yml@tox_linters
```

Comment:
Please don't use personal repos.

Reply:
This is how it was currently on the collection, I did not noticed that, I will update it